src/sentry/search/events/base.py

```diff
@@ -131,6 +133,16 @@ def _resolve_transaction_status(self, _: str) -> SelectType:
             "toUInt8", [self.column(TRANSACTION_STATUS_ALIAS)], TRANSACTION_STATUS_ALIAS
         )
 
+    def _resolve_error_unhandled_alias(self, _: str) -> SelectType:
+        return Function("notHandled", [], ERROR_UNHANDLED_ALIAS)
+
+    def _resolve_error_handled_alias(self, _: str) -> SelectType:
+        # Columns in snuba doesn't support aliasing right now like Function does.
+        # Adding a no-op here to get the alias.
+        return Function(
+            "cast", [self.column("error.handled"), "Array(Nullable(UInt8))"], ERROR_HANDLED_ALIAS
```

Comment:
lets stick with `toUint`
```suggestion
            "toUint8", [self.column("error.handled")], ERROR_HANDLED_ALIAS
```

Reply:
eugh, it doesn't work bc it's a Nullable(UInt8) so I can cast it to UInt8. And I cant do UInt8OrNull either bc apparently the contents need to be a string. This is basically the only thing tat seems to work

Reply:
Ah dang its fine then :+1: